app/models/navigation/Menus.scala

```diff
@@ -50,6 +50,12 @@ class Menus(implicit applicationFeatures: ApplicationFeatures) {
     } else None
   }
   
+  private[this] def scheduleLeaderElectionMenu(cluster: String) : Option[Menu] = {
+    if (applicationFeatures.features(KMScheduleLeaderElectionFeature)) {
+      Option("Schedule Leader Election".clusterMenu(cluster))
```

Comment:
Some

Reply:
This file is using `Option` everywhere so leaving this as-is for consistency.